src/viam/app/app_client.py

```diff
@@ -1369,10 +1369,11 @@ async def upload_module_file(self, module_file_info: Optional[ModuleFileInfo], f
         async with self._app_client.UploadModuleFile.open(metadata=self._metadata) as stream:
             await stream.send_message(request_module_file_info)
             await stream.send_message(request_file, end=True)
-            response = await stream.recv_message()
-            if not response:
-                await stream.recv_trailing_metadata()  # causes us to throw appropriate gRPC error.
-            return response.url
+            response: Union[UploadModuleFileRequest, None] = await stream.recv_message()
+            if response is not None:
+                return response.url
+            await stream.recv_trailing_metadata()  # causes us to throw appropriate gRPC error.
```

Comment:
It looks like this got inverted from how it was before? we want to `recv_trailing_metadata()` to throw an error, when `response` is `None`.

Reply:
Not sure why I changed it, but isn't this functionally the same?

Reply:
🤦 yes you're absolutely right, read it too quickly and thought the `if` statements were equivalent.

I do have a slight preference for either leaving it the way it was or updating the equivalent logic in `data_client.py` and `audo_input/client.py` just to all be consistent, but I don't feel strongly enough to hold off release.